test/parallel/test-util-inspect.js

```diff
@@ -588,8 +588,9 @@ assert.doesNotThrow(function() {
                      '{ a: 123, inspect: [Function: inspect] }');
 
   const subject = { a: 123, [util.inspect.custom]() { return this; } };
+  const UTC = 'util.inspect.custom';
```

Comment:
Oops. Just a typo. I meant UIC for `util.inspect.custom` But, from Wikipedia:
```
UIC may refer to:

In education:

Underwood International College, a department of Yonsei University
United International College, a liberal arts college in China
University of Illinois at Chicago, a public four-year university in Chicago, Illinois
In government:

Uganda Insurance Commission, a government agency
Unit Identification Code, an alphanumeric code used by the United States Department of Defense
In organisations:

International Union of Railways (Union Internationale des Chemins de fer), an international rail transport industry body
UEFA Intertoto Cup, an association football tournament for member clubs of UEFA
Union of Islamic Courts, an armed group in Somalia, now called the "Supreme Islamic Courts Council"
United Industrial Corporation Ltd, a Singapore real estate holding company owned by JG Summit Holdings
In computing:

Uranium Information Centre, a defunct website sponsored by uranium mining companies
User identification code, the user number of the Files-11 file system in the RSX-11 operating system
In other uses:

UIC (band), a punk rock band from Canada
UIC Flames, the intercollegiate athletic program of the University of Illinois at Chicago
```
What is a good name?